lib/interactive.js

```diff
@@ -103,6 +103,17 @@ function renderMessage(author, message) {
     if (message.body !== undefined && message.body !== "")
         msg += message.body;
 
+    if (message.sticker) {
+        const x = `${attsNo}`;
+
+        const label = message.sticker.label
+            ? message.sticker.label
+            : 'sticker';
+
+        msg += `${' [ '.red + label } ${  x.cyan  }${' ]'.red}`;
```

Comment:
I would add this line right after, so you can use `/v {attsNo}` to view the sticker in browser
```
atts[attsNo] = message.sticker.url;
```